charts/ingress-nginx/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 This file documents all notable changes to [ingress-nginx](https://github.com/kubernetes/ingress-nginx) Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+### 3.31.0
+
+- [#7117](https://github.com/kubernetes/ingress-nginx/pull/7092) Add startup probes and annotations for HPA 
```

Comment:
I’m not sure if this change can be included here, or if other changes need to be recorded.

Reply:
Then should I just bump the patch version: 

```
### 3.30.1

- [#7117](https://github.com/kubernetes/ingress-nginx/pull/7117) Add startup probes and annotations for HPA 
```